metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/dsl/functions/OrdinalFunctions.java

```diff
@@ -0,0 +1,75 @@
+package org.apache.metron.stellar.dsl.functions;
+
+import org.apache.metron.stellar.dsl.BaseStellarFunction;
+import org.apache.metron.stellar.dsl.Stellar;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OrdinalFunctions {
+
+    /**
+     * Stellar Function: MAX
+     * <p>
+     * Return the maximum value of a list of input values in a Stellar list
+     */
```

Comment:
It may be more clear to have MAX  | MIN  function for Numbers and a set of SORTED functions for Strings.